src/plugins/embeddable/public/lib/actions/edit_panel_action.ts

```diff
@@ -132,4 +133,11 @@ export class EditPanelAction implements Action<ActionContext> {
     const editUrl = embeddable ? embeddable.getOutput().editUrl : undefined;
     return editUrl ? editUrl : '';
   }
+
+  private getExplicitInput({ embeddable }: ActionContext): EmbeddableInput {
+    return (
+      (embeddable.getRoot() as Container)?.getInput()?.panels?.[embeddable.id]?.explicitInput ??
```

Comment:
does using `.panels` make this specific for dashboard embeddable container ?

Reply:
Panels is on the `ContainerInput` interface, so it should technically work with any container. 

This method `getExplicitInput` is meant to separate input inherited from parent from input that belongs to the embeddable, and without a 'container' parent, there is nothing to pass down input so the default case of just returning the embeddable's input works well.